### PR TITLE
npm workspaces

### LINKED ISSUE #23: Migrate from lerna/nx to npm workspace and add intelliJ run configurations for each package
We are currently using nx, lerna and subpackage (in examples). 

Now that npm supports workspaces we should migrate and use it as it reduces the level of complexity in our library.

The projects are deeply nested, e.g. the vue example resides in `source/examples/vue/six-app`. 

New proposed structure:
```
/packages  
 /ui-library  
 /ui-library-react 
 /ui-library-vue  
 /demo-app-angular  
 /demo-app-react  
 /demo-app-vue
```

### PR DESCRIPTION
**Pull request by ibirrer**
Use [npm workspaces](https://docs.npmjs.com/cli/v8/using-npm/workspaces) instead of lerna, nx and subpackage. Closes #23

### CODE REVIEW

File: .github/workflows/ci.yml
[the remark this answers is not in the document]
Comment (ibirrer) [reply .github/workflows/ci.yml:1]: I quickly checked if it really makes a difference for the build time - short answer: it doesn't. Looks like it is even faster without it. Note that the `cache: 'npm'` options (which is still there) already caches the npm internal cache. Additional caching node_modules doesn't make it significantly faster but makes the config more complicated.

File: package.json [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+  "name": "@six-group/workspace",
   "scripts": {
-    "buildCI": "node scripts/buildCI.js",
-    "publish": "node scripts/publish.js",
-    "postinstall": "lerna bootstrap",
-    "build": "lerna run build",
-    "version:nopush": "lerna version --no-push",
-    "clean": "lerna run clean",
+    "build": "npm run build --workspaces",
+    "build-lib": "npm run build                 -w packages/ui-library -w packages/ui-library-react -w packages/ui-library-vue",
```
[the remark this answers is not in the document]
Comment (ibirrer) [reply package.json:5]: I usually like to align lines which are completely the the same. Here, in particular, the workspaces `-w ... -w ...` in order to spot differences quickly:
```
    "build-lib": "npm run build                 -w packages/ui-library -w packages/ui-library-react -w packages/ui-library-vue",
    "publish-lib": "npm publish --access public -w packages/ui-library -w packages/ui-library-react -w packages/ui-library-vue",
```

[the remark this answers is not in the document]
Comment (colinscz) [reply package.json:5]: ok thanks for the explanation, that is fine by me :)

**Review by colinscz — APPROVED** [review]

**Comment by colinscz** [comment]
@ibirrer please update the commit message and add "closes #23 " that way the corresponding issue is automatically closed when this PR is merged

**Review by colinscz — APPROVED** [review]

**Pull request merged**